src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -73,18 +73,32 @@
 
         <tr class="heading">
             <td>{{ 'sylius.ui.item'|trans }}</td>
-            <td>{{ 'sylius.ui.total'|trans }}</td>
             <td>{{ 'sylius.ui.tax_total'|trans }}</td>
+            <td>{{ 'sylius.ui.total'|trans }}</td>
         </tr>
 
         {% for unit in creditMemo.units %}
         <tr class="item">
             <td>{{ unit.productName }}</td>
-            <td>{{ unit.total|sylius_format_money(creditMemo.currencyCode, creditMemo.localeCode) }}</td>
             <td>{{ unit.taxesTotal|sylius_format_money(creditMemo.currencyCode, creditMemo.localeCode) }}</td>
+            <td>{{ unit.total|sylius_format_money(creditMemo.currencyCode, creditMemo.localeCode) }}</td>
         </tr>
         {% endfor %}
 
+        {% if creditMemo.taxItems|length > 0 %}
+            <tr class="heading">
+                <td colspan="2">{{ 'sylius.ui.tax'|trans }}</td>
```

Comment:
Maybe `Taxes` not `Tax`? There can be multiple of them

Reply:
I wanted to be consistent with `InvoicingPlugin`